scripts/create-example-tests.js

```diff
@@ -1144,9 +1147,24 @@ function validateTest(testParsed, data, { addTestError = () => {} } = {}) {
  * @param {Queryable<AriaATParsed.Key>} data.key
  * @param {Queryable<{name: string, path: string}>} data.example
  * @param {Queryable<{at: string, mode: string, render: function({key: *}): string}>} data.modeInstructionTemplate
+ * @param {string} testPlanDirectory
  * @returns {AriaATFile.CollectedTest}
  */
-function collectTestData({ test, command, key, example, modeInstructionTemplate }) {
+function collectTestData(
+  { test, command, key, example, modeInstructionTemplate },
+  testPlanDirectory
+) {
+  const referencePageLocation = example.where({
+    name: test.setupScript ? test.setupScript.name : '',
+  });
+
+  if (!referencePageLocation) {
+    console.error(
+      `ERROR: Setup script ${test.setupScript.name} found in "${testPlanDirectory}/tests.csv" for "test id ${test.testId}: ${test.task}" not defined in "${testPlanDirectory}/reference".`
+    );
+    process.exit(1);
```

Comment:
I don't think this error is critical enough to stop the process. Any error recorded by `addTestError`, `addCommandError`, etc, will be reported together at the end of processing this test plan, an `isSuccessfulRun: false` will be reported to `create-all-tests`, which in response will include its failure in the summary and `exit(1)`.